docs/README.md

```diff
@@ -1,154 +1,233 @@
+After a release, documentation updates are continually merged into `master` as they occur. This work includes new documentation for forthcoming features, bug fixes, and other updates. 
 
-    $ make docs
-    .... (lots of output) ....
-    docker run --rm -it  -e AWS_S3_BUCKET -p 8000:8000 "docker-docs:master" mkdocs serve
-    Running at: http://0.0.0.0:8000/
-    Live reload enabled.
-    Hold ctrl+c to quit.
+Periodically, the Docker maintainers update `docs.docker.com` between official releases of the Docker product. They do this by cherry-picking commits from `master`, merging them into `docs`,  and the publishing the result.
 
-If you have any issues you need to debug, you can use `make docs-shell` and then
-run `mkdocs serve`
+In the rare case where a change is not forward-compatible, changes may be made
+on other branches by special arrangement with the Docker maintainers.
 
-## Testing the links
+### Quickstart for documentation contributors
 
-You can use `make docs-test` to generate a report of missing links that are referenced in
-the documentation - there should be none.
+If you are a new or beginner contributor, we encourage you to read through the
+ [our detailed contributors
+guide](https://docs.docker.com/project/who-written-for/). The guide explains in detail with examples how to contribute. If you are an experienced contributor this quickstart should be enough to get you started.
 
-## Adding a new document
+The following is the essential workflow for contributing to the documentation:
 
-New document (`.md`) files are added to the documentation builds by adding them
-to the menu definition in the `docs/mkdocs.yml` file.
+1. Fork the `docker/docker` repository.
 
-## Style guide
+2. Clone the repository to your local machine.
 
-If you have questions about how to write for Docker's documentation (e.g.,
-questions about grammar, syntax, formatting, styling, language, or tone) please
-see the [style guide](sources/contributing/docs_style-guide.md). If something
-isn't clear in the guide, please submit a PR to help us improve it.
+3. Select an issue from `docker/docker` to work on or submit a proposal of your own.
 
-## Working using GitHub's file editor
+4. Create a feature branch from `master` in which to work.
 
-Alternatively, for small changes and typos you might want to use GitHub's built-
-in file editor. It allows you to preview your changes right on-line (though
-there can be some differences between GitHub Markdown and [MkDocs
-Markdown](http://www.mkdocs.org/user-guide/writing-your-docs/)).  Just be
-careful not to create many commits. And you must still [sign your
-work!](../CONTRIBUTING.md#sign-your-work)
+	By basing from `master` your work is automatically included in the next
+	release. It alos allows docs maintainers to easily cherry-pick your changes
+	into the `docs` release branch. 
 
-## Branches
+4. Modify existing or add new `.md` files to the `docs/sources` folder.
```

Comment:
Should we be consistent wrt the term for "folder" / "directory"? I think "folder" (as a term) is only used on OS X and "directory" on all other platforms.

(Here, and couple of other places in this document)